src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -149,3 +152,15 @@ function shouldShowPropertyName(schemaNode: SchemaNode) {
     (schemaNode.subpath[0] === 'properties' || schemaNode.subpath[0] === 'patternProperties')
   );
 }
+
+function anyOfOrOneOf(schemaNode: SchemaNode) {
+  if (schemaNode.fragment.anyOf !== undefined) {
+    return '(Any of)';
+  }
+
+  if (schemaNode.fragment.oneOf !== undefined) {
+    return '(One of)';
+  }
+
+  return undefined;
+}
```

Comment:
dead code now, nah? 
```suggestion
function anyOfOrOneOf(schemaNode: SchemaNode) {
  if (schemaNode.fragment.anyOf !== undefined) {
    return '(Any of)';
  }

  if (schemaNode.fragment.oneOf !== undefined) {
    return '(One of)';
  }

  return undefined;
}
```

or did we intend to use it in `src/components/SchemaRow/TopLevelSchemaRow.tsx`?

Reply:
:ope: my bad